src/Core/WinSWCore/Configuration/DefaultSettings.cs

```diff
@@ -46,13 +49,54 @@ public sealed class DefaultWinSWSettings : IWinSWConfiguration
         public bool Interactive => false;
 
         // Logging
-        public string LogDirectory => Path.GetDirectoryName(ExecutablePath)!;
-        public string LogMode => "append";
+        public Log Log { get => new LogDefaults(); }
+        public string LogDirectory => DefaultLogSettings.Directory;
+        public string LogMode => DefaultLogSettings.Mode;
+
+        public bool OutFileDisabled => Log.OutFileDisabled;
+        public bool ErrFileDisabled => Log.ErrFileDisabled;
+        public string OutFilePattern => Log.OutFilePattern;
+        public string ErrFilePattern => Log.ErrFilePattern;
+
+
+        public class LogDefaults : Log
+        {
+            readonly DefaultWinSWSettings defaults;
+            readonly ServiceDescriptorYaml serviceDescriptorYaml;
```

Comment:
What is this magic? Why do default settings need to depend on the YAML configuration? Some refatoring is needed AFAICT

Reply:
Updated this.